test/nri/plugin.go

```diff
@@ -209,6 +209,26 @@ func (p *plugin) RunPodSandbox(_ context.Context, pod *api.PodSandbox) error {
 	return nil
 }
 
+func (p *plugin) UpdatePodSandbox(_ context.Context, pod *api.PodSandbox) error {
+	if !p.inNamespace(pod.GetNamespace()) {
+		return nil
+	}
+
+	p.Lock()
+	defer p.Unlock()
+
+	p.pods[pod.GetId()] = pod
```

Comment:
nit: this should not be done here... we should already have a pod (since it has been started and has not been stopped and removed yet). What we instead could do here is look it up and patch in the updated overhead and resources...